.circleci/config.yml

```diff
@@ -25,14 +33,77 @@ jobs:
           apk update && apk upgrade
           apk add --no-cache git openssh make gcc g++ python
       - run:
-          name: Install dev dependencies
-          command: npm install
+          name: Install Dependencies
+          command: |
+            npm install
+            npm run lerna-reset
       - run:
-          name: Run linter
+          name: Run Linter
           command: npm run lint
       - run:
-          name: Bootstrap lerna
-          command: npm run lerna-reset
-      - run:
-          name: Run unit tests
+          name: Run Unit Tests
           command: npm run test
+
+  integration-test:
+    machine:
+      image: ubuntu-1604:201903-01 
+    steps:
+      - checkout
+      - run:
+          name: Setup Machine
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v12.16.1
+            nvm alias default v12.16.1
+            curl -L https://github.com/docker/compose/releases/download/1.25.4/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+            chmod +x ~/docker-compose
+            sudo mv ~/docker-compose /usr/local/bin/docker-compose
+            docker-compose --version
+      - run:
+          name: Install Dependencies
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            npm install
+            npm run lerna-reset
+      - run:
+          name: Setup elixir-omg
+          command: |
+            mkdir -p ~/.ssh/
+            echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config
+            git clone git@github.com:omisego/elixir-omg.git ~/elixir-omg
+            cd ~/elixir-omg
+            SNAPSHOT=SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_120 make init_test
```

Comment:
I think people from elixir-omg mentioned that instead of `init_test`, it is better to use `make init-contracts`
(They are just proxies of each other)

Reply:
Yup thanks! Yeah this command is now being used outside tests too so I've renamed it `make init-contracts`, the init_test is now just an alias and to be removed in the future.